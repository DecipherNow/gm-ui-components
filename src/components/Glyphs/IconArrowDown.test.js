import React from "react";
import { shallow } from "enzyme";

import IconArrowDown from "./IconArrowDown.svg";

describe("IconArrowDown", () => {
  it("matches snapshot", () => {
    const aIconArrowDown = shallow(<IconArrowDown />).find("IconArrowDown.svg");
    expect(aIconArrowDown).toMatchSnapshot();
  });
});