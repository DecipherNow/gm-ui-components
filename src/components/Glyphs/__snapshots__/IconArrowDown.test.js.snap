// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconArrowDown matches snapshot 1`] = `
<Icon
  backgroundColor="currentColor"
  backgroundOpacity={1}
  backgroundSizeRatio={1}
  borderColor="currentColor"
  borderOpacity={1}
  borderWidth={1}
  glyph=""
  glyphColor="currentColor"
  glyphName="ArrowDown"
  glyphSizeRatio={1}
>
  <polygon
    id="ArrowDown"
    points="9.49999999 8.49999999 14.5 12.5 9.49999999 16.5"
    transform="translate(12.000000, 12.500000) rotate(90.000000) translate(-12.000000, -12.500000)"
  />
</Icon>
`;
